app/src/main/java/com/example/andresarango/aughunt/challenge/challenges_adapter/ChallengeViewholder.java

```diff
@@ -0,0 +1,41 @@
+package com.example.andresarango.aughunt.challenge.challenges_adapter;
+
+
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.example.andresarango.aughunt.R;
+import com.example.andresarango.aughunt.challenge.Challenge;
+import com.example.andresarango.aughunt.challenge.challenge_dialog_fragment.DialogFragmentListener;
+
+class ChallengeViewholder<T> extends RecyclerView.ViewHolder {
```

Comment:
i.e., not important

http://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak